### PR TITLE
[PATCH v2] test: queue_perf: minor improvements

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

File: test/performance/odp_queue_perf.c [hunk 1]
```diff
@@ -190,12 +190,12 @@ static int create_queues(test_global_t *global)
 		event[i] = ODP_EVENT_INVALID;
```
Comment (TuomasTaipale) [inline test/performance/odp_queue_perf.c:196]: There still seems to be quite a few error-like prints using `printf()`, swap those as well?

(E.g. 
```c
		printf("Too many queues %u. Test maximum %u.\n",
		       test_options->num_queue, MAX_QUEUES);

...
			printf("Max queues supported %u\n",
			       queue_capa.plain.max_num);
```
)

File: test/performance/odp_queue_perf.c [hunk 2]
```diff
@@ -608,25 +608,25 @@ int main(int argc, char **argv)
...
 	if (shm == ODP_SHM_INVALID) {
-		ODPH_ERR("Error: Shared mem reserve failed.\n");
+		ODPH_ERR("Shared mem reserve failed.\n");
 		exit(EXIT_FAILURE);
 	}
 
 	global = odp_shm_addr(shm);
 	if (global == NULL) {
-		ODPH_ERR("Error: Shared mem alloc failed\n");
+		ODPH_ERR("Shared mem alloc failed\n");
```
Comment (TuomasTaipale) [inline test/performance/odp_queue_perf.c:636]: Old code, but the "alloc" here for `odp_shm_addr()` is a bit confusing.

File: test/performance/odp_queue_perf.c [hunk 3]
```diff
@@ -153,6 +153,11 @@ static int parse_options(int argc, char *argv[], test_options_t *test_options)
 		return -1;
 	}
 
+	if (test_options->single && test_options->num_cpu != 1) {
+		printf("Using single producer/consumer queue(s) with multiple workers not supported.\n");
```
Comment (TuomasTaipale) [inline test/performance/odp_queue_perf.c:157]: Error-like, so `ODPH_ERR()`?

File: test/performance/odp_queue_perf.c (hunk 1 above)
Comment (MatiasElo) [reply test/performance/odp_queue_perf.c:196]: Change in V2.

File: test/performance/odp_queue_perf.c (hunk 2 above)
Comment (MatiasElo) [reply test/performance/odp_queue_perf.c:636]: Message clarified in V2.

File: test/performance/odp_queue_perf.c (hunk 3 above)
Comment (MatiasElo) [reply test/performance/odp_queue_perf.c:157]: Changed in V2.

**Pull request merged**